### PR TITLE
Add a test for marshalling nil double->Double

### PR DESCRIPTION
**Pull request by heyagosper**

### CODE REVIEW

**Comment by lbodor** [comment]
:+1:  I love small PRs!

File: src/test/java/au/gov/ga/geodesy/support/marshalling/moxy/GeodesyMLMoxyTest.java [hunk 1]
```diff
@@ -39,6 +40,32 @@ public void unmarshal() throws Exception {
...
     @Test
+    public void unmarshalWithNullNumericFields() throws Exception {
+        Reader input = new InputStreamReader(Thread.currentThread()
+                .getContextClassLoader()
+                .getResourceAsStream("MOBS-null-numerics.xml"));
+
+        GeodesyMLType geodesyML = marshaller.unmarshal(input, GeodesyMLType.class).getValue();
+        List<JAXBElement<?>> els = geodesyML.getElements();
+        Assert.assertNotNull(els);
+        Assert.assertNotEquals(0, els.size());
```
Comment (oehm-smith) [inline src/test/java/au/gov/ga/geodesy/support/marshalling/moxy/GeodesyMLMoxyTest.java:51]: What happened to Hamcrest? :)
